package.json

```diff
@@ -53,6 +53,7 @@
     "eslint-plugin-import-x": "~4.10.0",
     "eslint-plugin-no-relative-import-paths": "~1.6.1",
     "prettier": "3.5.3",
-    "prettier-plugin-tailwindcss": "0.6.11"
+    "prettier-plugin-tailwindcss": "0.6.11",
+    "unrs-resolver": "^1.7.0"
```

Comment:
the `unrs-resolver` dependency fixes an issue I was hitting in CI:
```
Oops! Something went wrong! :(

ESLint: 9.23.0

Error: Failed to load native binding
```

(related: https://github.com/unrs/unrs-resolver/issues/56)

I must admit that I haven't really understood what the issue is here 🤔 if anyone has any idea on how to maybe better address/investigate it I'm more than happy to look into that 🙂

Reply:
Do we actively use the dep? 

If not, maybe add it to overrides _for now_, and we can investigate further?

Reply:
> Do we actively use the dep?

No

> If not, maybe add it to overrides for now, and we can investigate further?

Good call, yeah, sorry I keep forgetting the npm has overrides 😓 

PS: I can keep investigating further but my worry is that this might just be fixed by the pnpm migration or if a solution is found for npm such can be invalidated by the pnpm migration 🤔

Reply:
Possibly, yes. Let's wait the pnpm PR to get merged first, before this one. Just to ensure all of this also works on pnpm :sweat_smile: